packages/flutter_tools/lib/src/commands/create_base.dart

```diff
@@ -130,17 +131,28 @@ abstract class CreateBase extends FlutterCommand {
   @protected
   void addPlatformsOptions({String customHelp}) {
     argParser.addMultiOption('platforms',
-        help: customHelp ?? _kDefaultPlatformArgumentHelp,
-        defaultsTo: _kAvailablePlatforms,
-        allowed: _kAvailablePlatforms);
+      help: customHelp ?? _kDefaultPlatformArgumentHelp,
+      defaultsTo: <String>[
+        ..._kAvailablePlatforms,
```

Comment:
make sure winuwp is conditionally added so it doesn't show up as an option unless you've already opted in